src/jit/gentree.h

```diff
@@ -2429,6 +2524,14 @@ struct GenTreeCall final : public GenTree
     }
 #endif // !LEGACY_BACKEND
 
+    // Returns true if the call has retBuf argument
+    bool HasRetBufArg() const { return (gtCallMoreFlags & GTF_CALL_M_RETBUFFARG) != 0; }
```

Comment:
I would envision that HasRetBufArg would be something you would query of the ReturnTypeDesc